### PR TITLE
[WebRTC] Change Stats enum value and add new enum value

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

- Change Stats enum value
- Add new value in WebRTCStatisticsCategory enum

### API Changes ###

 - TCSACR-535

### CODE REVIEW

**Comment by hsgwon** [comment]
API 변경에 대한 결재 진행 중.

**Review by aferin — APPROVED** [review]

**Comment by hsgwon** [comment]
> API 변경에 대한 결재 진행 중.

결재 완료. ACR페이지에 결재 메일 첨부

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs:691]: that summary is attached to the Port property, the Local ID is at line 674

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs:534]: changing of existing enum values are not normally allowed, an argument can be made that this holds the purpose of the "All" unchanged

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs:549]: changing the enum values are normally allowed, I will approve the PR only after confirming it is ok

**Review by wiertel — CHANGES_REQUESTED** [review]

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs:534]: I've got approval for change from PM part.

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs:691]: I'll fix it.

**Review by wiertel — APPROVED** [review]

**Pull request merged**
